src/PythonQt.cpp

```diff
@@ -525,6 +536,7 @@ void PythonQtPrivate::createPythonQtClassWrapper(PythonQtClassInfo* info, const
     PythonQtClassInfo* outerClassInfo = lookupClassInfoAndCreateIfNotPresent(outerClass);
     outerClassInfo->addNestedClass(info);
   } else {
+    Py_INCREF(pyobj);
     PyModule_AddObject(pack, info->className(), pyobj);
```

Comment:
PyModule_AddObject only decrements the reference count on success, so you should check the return code.

Reply:
Yes, I saw this, there are many more such calls and I thought i leave it up for the next round of fixing, but now I fixed all of them by adding a wrapper function to do the job.